model/src/main/kotlin/utils/Extensions.kt

```diff
@@ -157,7 +157,7 @@ fun Identifier.getPurlType() =
         "composer" -> PurlType.COMPOSER
         "conan" -> PurlType.CONAN
         "crate" -> PurlType.CARGO
-        "godep", "gomod" -> PurlType.GOLANG
+        "go" -> PurlType.GOLANG
```

Comment:
same here, maybe it would be worth to keep the old mapping for backwards compatibility

Reply:
I'll make a follow-up PR so this can be discussed separately.